backend/service/k8s/statefulset.go

```diff
@@ -67,12 +67,28 @@ func ProtoForStatefulSet(cluster string, statefulSet *appsv1.StatefulSet) *k8sap
 	if clusterName == "" {
 		clusterName = cluster
 	}
-	return &k8sapiv1.StatefulSet{
+	k8sStateful := &k8sapiv1.StatefulSet{
 		Cluster:     clusterName,
 		Namespace:   statefulSet.Namespace,
 		Name:        statefulSet.Name,
 		Labels:      statefulSet.Labels,
 		Annotations: statefulSet.Annotations,
+		Status:      ProtoForStatus(statefulSet.Status),
+	}
+
+	if !statefulSet.CreationTimestamp.IsZero() {
+		// Convert Unix Timestamp to milliseconds
+		k8sStateful.CreationTimeMillis = statefulSet.CreationTimestamp.UnixNano() / 1e6
+	}
+
+	return k8sStateful
+}
+
+func ProtoForStatus(status appsv1.StatefulSetStatus) *k8sapiv1.StatefulSet_Status {
+	return &k8sapiv1.StatefulSet_Status{
+		Replicas:        uint32(status.Replicas),
```

Comment:
I assume status will never be `nil`?

Reply:
Status is the current status of Pods in this StatefulSet so the status and number of replicas can never be nil similar to deployment status